src/subscriptions/launcher.rs

```diff
@@ -39,6 +39,7 @@ pub fn subscription<I: 'static + Hash + Copy + Send + Sync>(
     })
 }
 
+#[cfg(feature = "console")]
 /// Initializes pop-launcher if it is not running, and returns a handle to its client.
 async fn client_request<'a>(
```

Comment:
It doesn't work because it's disabling the pop-launcher

Reply:
Can confirm -- it compiles and runs fine